src/MojiraBot.ts

```diff
@@ -133,6 +133,18 @@ export default class MojiraBot {
 					}
 				}
 
+                                if ( BotConfig.request.logChannel ) {
+                                        try {
+                                                const logChannel = await DiscordUtil.getChannel( BotConfig.request.logChannel );
+                                                if ( logChannel instanceof TextChannel ) {
+                                                        const messages: ChannelLogsQueryOptions = { limit: 100 };
+                                                        await logChannel.messages.fetch( messages );
```

Comment:
```suggestion
                                                        await logChannel.messages.fetch( { limit: 100 } );
```
The variable name `messages` sound weird to me, and we could actually just inline the setting here afaict.